### PR TITLE
Add nullable tags for SCCs, take 2

### PR DESCRIPTION
**Pull request by sallyom**
Adding nullable tags for SCCs to convert SCCs to CRD
see: https://github.com/openshift/cluster-config-operator/pull/41

### CODE REVIEW

**Comment by sallyom** [comment]
@damemi in the cluster-config-op PR, I'm seeing `Available: v1.security.openshift.io is not ready: 503"` in openshift-apiserver-operator logs.  I'm assuming it's validation error but please tell me if i'm wrong (also tell me exactly where I'd see the openAPI validation errors manifest in logs, pleeeese)

File: security/v1/types.go [hunk 1]
```diff
@@ -24,6 +24,7 @@ type SecurityContextConstraints struct {
 	metav1.TypeMeta `json:",inline"`
 	// Standard object's metadata.
 	// More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata
+	// +nullable
```
Comment (soltysh) [inline security/v1/types.go]: I don't think we want `nullable` on `ObjectMeta`.

File: security/v1/types.go [hunk 2]
```diff
@@ -271,6 +277,7 @@ type SecurityContextConstraintsList struct {
 	metav1.TypeMeta `json:",inline"`
 
 	// More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata
+	// +nullable
```
Comment (soltysh) [inline security/v1/types.go]: Ditto.

File: security/v1/types.go (hunk 1 above)
Comment (sallyom) [reply security/v1/types.go]: right, fixed.

File: security/v1/types.go (hunk 2 above)
Comment (sallyom) [reply security/v1/types.go]: fixed

**Pull request merged**